src/celestia/qt/qtappwin.cpp

```diff
@@ -479,6 +469,11 @@ CelestiaAppWindow::init(const CelestiaCommandLineOptions& options)
     timer->start();
 }
 
+void
+CelestiaAppWindow::startAppCore()
```

Comment:
just `start`?

Reply:
Since there's already a constructor and an `init` method in terms of things that begin the object, I prefer to be slightly more explicit about precisely what this thing is starting.